lib/rules/selector-max-specificity/__tests__/index.mjs

```diff
@@ -179,6 +182,35 @@ testRule({
 			endLine: 1,
 			endColumn: 11,
 		},
+		{
+			code: '.foo { .bar {} }',
+			message: messages.expected('.bar', '0,1,0', '.foo .bar'),
+			line: 1,
+			column: 8,
+			endLine: 1,
+			endColumn: 12,
+		},
+		{
+			code: '#foo, .bar { & .baz {} }',
+			message: messages.expected('& .baz', '0,1,0', ':is(#foo,.bar) .baz'),
```

Comment:
[suggestion] This `message` expectation seems unnecessary, thanks to `warnings` within the same case:

```suggestion
```